lib/cfn-nag/custom_rules/IamRoleAdministratorAccessPolicyRule.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class IamRoleAdministratorAccessPolicyRule < BaseRule
+  def rule_text
+    'IAM role should not have AdministratorAccess policy'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W25'
```

Comment:
W25 used to be something i forgot... probably something with security groups that got collapsed.  can you bump this to something higher? W38 maybe?